matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/model/MXKey.kt

```diff
@@ -38,14 +38,18 @@ data class MXKey(
         /**
          * signature user Id to [deviceid][signature]
          */
-        private val signatures: Map<String, Map<String, String>>
+        private val signatures: Map<String, Map<String, String>>,
+
+        private val rawMap: JsonDict
```

Comment:
Can you add doc like other fields please?

Reply:
added